templates/firefox-ios.xliff

```diff
@@ -3915,6 +4002,197 @@ Private Tab</source>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.ReaderView.Subtitle.v131" xml:space="preserve">
+        <source>Reader View</source>
+        <note>On the main menu, a string below the Tool submenu tiitle, indicating what kind of tools are available in that menu. This string is for the Reader View tool.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.ReportBrokenSite.Subtitle.v131" xml:space="preserve">
+        <source>Report</source>
+        <note>On the main menu, a string below the Tool submenu tiitle, indicating what kind of tools are available in that menu. This string is for the Report Broken Site tool.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.ReportBrokenSite.Title.v131" xml:space="preserve">
+        <source>Report Broken Site</source>
+        <note>On the main menu, the title for the action that will take the user to the site where they can report a broken website to our web compatibility team.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.Share.Subtitle.v131" xml:space="preserve">
+        <source>Share</source>
+        <note>On the main menu, a string below the Tool submenu tiitle, indicating what kind of tools are available in that menu. This string is for the Report Share tool.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.Share.Title.v131" xml:space="preserve">
+        <source>Share</source>
+        <note>On the main menu, the title for the action that will take the user to the Share module in the application.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.Zoom.Subtitle.v131" xml:space="preserve">
+        <source>Zoom</source>
+        <note>On the main menu, a string below the Tool submenu title, indicating what kind of tools are available in that menu. This string is for the Zoom tool.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.Zoom.Title.v131" xml:space="preserve">
+        <source>Zoom (%@)</source>
+        <note>On the main menu, in the tools submenu, the title for the menu component that indicates the current zoom level. Placeholder is for the current zoom level percentage.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.TabsSection.NewPrivateTab.Title.v131" xml:space="preserve">
+        <source>New Private Tab</source>
+        <note>On the main menu, the title for the action that will create a new private tab.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.TabsSection.NewTab.Title.v131" xml:space="preserve">
+        <source>New Tab</source>
+        <note>On the main menu, the title for the action that will create a new, non-private, tab.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.ToolsSection.FindInPage.Title.v131" xml:space="preserve">
+        <source>Find in Page...</source>
```

Comment:
@adudenamedruby Linter caught this, should be an ellipsis (vs three dots). Thanks!